platforms/android/app/src/main/assets/www/js/index.js

```diff
@@ -19,115 +19,115 @@
-        userId: "etLiIhOADD3F7EUZgdDackmmZbRji5",
-        signature: "WvbubBK85F6yDYddG8wb9iVCmjbZQqaDeVeGKsQoe1c="
-    };
-
-    let theme = {
-        "type": "light",
-        "detailColor1": "048b54",
-        "detailColor2": "048b54",
-        "detailColor3": "048b54",
-        "detailColor4": "048b54",
-        "detailColor5": "048b54",
-        "navigationBarTintColor": "048b54",
-        "navigationBarTitleImage": "Bank"
-    };
-
-    let paymentMethods = [
-        {
-        uniqueIdentifier: "IBAN_1",
-        name: "My Bank",
-        detailDescription: "Balance: 20CHF",
-        funds: 20.0,
-        image: "Bank"
-        },
-    ];
-
-    sonect.init(credentials, paymentMethods, theme);
-
-    sonect.present(
-        function(uniqueIdentifier, balanceCallback) {
-            let balance = {
-                uniqueIdentifier: uniqueIdentifier,
-                value: "100.00",
-                currency: "CHF"
-            };
-            balanceCallback(balance);
-        },
-        function(uniqueIdentifier, value, currency, paymentCallback) {
-            //To process payment without leaving the SDK, follow this code path
-            // let paymentReference = {
-            //     uniqueIdentifier: uniqueIdentifier,
-            //     paymentReference: "PAYMENT_REFERENCE"
-            // };
-            // paymentCallback(paymentReference);
-
-            //Alternatively, hide the SDK, and call sonect.pay when you have obtained the payment reference
-            sonect.hide(null, null);
-            app.paymentUniqueIdentifier = uniqueIdentifier;
-        },
-        function(msg) {
-            document
-            .getElementById('deviceready')
-            .querySelector('.received')
-            .innerHTML = msg;
-        },
-        function(err) {
-            document
-            .getElementById('deviceready')
-            .innerHTML = '<p class="event received">' + err + '</p>';
-        }
-    )
-},
-
-paySonect: function() {
-    let paymentReference = {
-        value: "20.00",
-        currency: "CHF",
-        uniqueIdentifier: app.paymentUniqueIdentifier,
-        paymentReference: "PAYMENT_REFERENCE"
-    };
-
-    sonect.presentTransaction(paymentReference, 
-        function(msg) {
-        },
-        function(err) {
-        }
-    );
-}
+    receivedEvent: function (id) {
+        var parentElement = document.getElementById(id);
+        var listeningElement = parentElement.querySelector('.listening');
+        var receivedElement = parentElement.querySelector('.received');
+
+        listeningElement.setAttribute('style', 'display:none;');
+        receivedElement.setAttribute('style', 'display:block;');
+
+        console.log('Received Event: ' + id);
+    },
+
+    openSonect: function () {
+        let credentials = {
+            token: "NDBiZDFjNzAtNzk4OC0xMWVhLTgzMWEtOWJlOWFiMzY1MjY5OmZhZTAyNGFkMmE5ZDRkMDI0ZjUxN2VmOTg5MTA3MjFiM2E0YWY5YzZmZjk4Y2M1N2FlOWMzZmEyMWMzMTcxYzY=",
```

Comment:
@markohlebar we changed creds from sonect's to BNL from dev because we have some if-then about sonect partner on BE. Should work for iOS as well.

Reply:
Yup, should work!